src/index/CMakeLists.txt

```diff
@@ -14,4 +13,10 @@ add_library(index
         PrefixHeuristic.cpp PrefixHeuristic.h
         CompressedRelation.h CompressedRelation.cpp)
 
-target_link_libraries(index parser ${STXXL_LIBRARIES} ${ICU_LIBRARIES} absl::flat_hash_map absl::flat_hash_set zstd)
+
+find_package(OpenMP)
+if(OpenMP_CXX_FOUND)
+    target_link_libraries(index parser ${STXXL_LIBRARIES} ${ICU_LIBRARIES} absl::flat_hash_map absl::flat_hash_set zstd OpenMP::OpenMP_CXX)
+else()
+    target_link_libraries(index parser ${STXXL_LIBRARIES} ${ICU_LIBRARIES} absl::flat_hash_map absl::flat_hash_set zstd)
+endif()
```

Comment:
I'd rather leave this out in this commit, 
And solve it on the project level with global enabling
of parallelism.